pkgs/development/python-modules/PyLD/default.nix

```
@@ -29,6 +29,13 @@ buildPythonPackage {
     sha256 = "1bmpz4s6j7by6l45wwxy7dn7hmrhxc26kbx2hbfy41x29vbjg6j9";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/digitalbazaar/pyld/pull/93.diff";
```

Please use the specific commit patch
```suggestion
      url = "https://github.com/digitalbazaar/pyld/commit/ec4bd907b7fbd566e1f6c76d09676e4900e3dcd0.patch";
```

---

resolved